CHANGELOG.md

```diff
@@ -24,6 +24,10 @@
   or other error.  
   [Martin Redington](https://github.com/mildm8nnered)
   [#6052](https://github.com/realm/SwiftLint/issues/6052)
+* Fix false positives of `redundant_discardable_let` rule in `@ViewBuilder` functions,
```

Comment:
Needs an empty line above.

Also I think this should go into the "Enhancements" section.

Reply:
Added empty line and moved to enhancements section here.
https://github.com/realm/SwiftLint/pull/6075/commits/3da5c81f8282731a88fe33c5349c8fa8939fd006